### ISSUE TITLE
PK-Parameter in Time Profile Analysis - multiple questions

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/PK-Sim#2020]
While going through the documentation (https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-analysis-view) and comparing it with the actual output of the pk-parameters in PK-Sim, I realized I have multiple questions.

Following use case: I have a model with two compounds - Esomeprazole and R-Omeprazole (these are the two enantiomeres of Omeprazole). A container observer for the molecule Esomeprazole calculates the concentration of the sum of Esomeprazole and R-Omeprazole in Pervipheral Venous Blood Plasma. This observer is called "Omeprazole Plasma (Peripheral Venous Blood)"

In an iv simulation (both enantiomeres are administered via iv route), I select only the observer "Omeprazole Plasma (Peripheral Venous Blood)" for the output, but not the concentrations of the single compounds. When I open the PK-Analysis, I see this:

![image](https://user-images.githubusercontent.com/25903040/146546909-2cef7908-5976-4303-b579-92585eee15fa.png)

1) For the general understanding: the PK-parameters in the lower table are calculated for the selected outputs (and observed data, if available), and the PK-parameters in the upper table are calculated for all molecules in the model, even if I do not select their outputs?

2) How are the "_norm"-parameters (e.g. AUC_tEnd_norm) calculated in this case? For the observer which sums up concentrations of two compounds, I assume it only accounts for the dose of the molecule for which the observer is calculated (Esomeprazole in this case). If so, the calculated value is not correct. In fact, allowing user-defined observers introduces a problem here.

3) Why is "AUC Ratio (AUCR)" calculated for the compounds only, and not for the observer? Here actually it could be calculated (as opposed to 2) ). Same for C_max ratio.

4) I cannot find parameter "Total body clearance".

5) Instead, I see parameter  "Total body clearance/F", which I cannot find in the documentation at all.

6) Also parameters "Vd(plasma)/F" and "Vss(plasma)/F", which should not appear for the iv simulation.

@msevestre @Yuri05 @StephanSchaller

**Comment by PavelBal** [comment]
Now, a po simulation. Same setting - both enantiomeres are administered, and the observer for the total concentration is selected as output.

![image](https://user-images.githubusercontent.com/25903040/146549962-681da415-6cea-43a7-a744-e99ef7ae6104.png)

7) How are the parameters "Vss(plasma)/F", "Vd(plasma)/F", "Total plasma clearance/F", and "Total Body Clearance/F" calculated? I did not press "Calculate Bioavailability" so far, so which values does the calculation use?

8) The parameter "Total body clearance/F" cannot be found in the documentation.

9) After calculating bioavailability, I see this:

![image](https://user-images.githubusercontent.com/25903040/146550669-63062128-73ab-46bc-be3d-b83fa3a98e30.png)


Could it be, that the "/F" and not "/F" captions are mixed-up?

10) Why, after calculation bioavailability, the non- /F parameters (which are labeled as /F but probably wrong...) are not shown any more?

**Comment by PavelBal** [comment]
Complex administration protocol: a compound is administered as iv at time 0, and as po at time 120h.

11) Why cannot the "Fraction absorbed" be calculated?
12) Same question for "Vss (plasma)", "Vd (plasma)", and "Total plasma clearance"
13) AUC_tEnd, AUC_inf, %AUC(tlast-inf), AUC_tEnd_norm, AUC_inf_norm, C_tEnd are not calculated (though stated in documenation as "for all simulations")

**Comment by msevestre** [comment]
Some quick feedback:
The definition of the parameter/F (for instance Total plasma clearance/F) does not require F. The way it is calculated is based on the IV définition. If you want to know the real value, you need to multiply by F
That's why Total plasma clearance is only available once you have the bioavailability 
Value of being P/F x F 
So there is no inversion of parameters whatsoever

**Comment by msevestre** [comment]
>For the general understanding: the PK-parameters in the lower table are calculated for the selected outputs (and observed data, if available), and the PK-parameters in the upper table are calculated for all molecules in the model, even if I do not select their outputs?

This is correct. Some outputs are ALWAYS used, even if not selected. Those are 
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Core/Services/SimulationPersistableUpdater.cs#L35

* Venous Blood Plasma
* Peripheral Venous Blood Plasma
* F abs oral 

>How are the "_norm"-parameters (e.g. AUC_tEnd_norm) calculated in this case? For the observer which sums up concentrations of two compounds, I assume it only accounts for the dose of the molecule for which the observer is calculated (Esomeprazole in this case). If so, the calculated value is not correct. In fact, allowing user-defined observers introduces a problem here.

I think this is a bug. I need to think about it

>Why is "AUC Ratio (AUCR)" calculated for the compounds only, and not for the observer? Here actually it could be calculated (as opposed to 2) ). Same for C_max ratio.

This is calculated for all compounds (and only once using peripheral venous blood except for mouse and rat where it is using venous blood). This is how it was specified. We have 100s of observers so I am not quite sure what you mean with calculated for observer

>I cannot find parameter "Total body clearance".

This should be Total body clearance/F

>Instead, I see parameter "Total body clearance/F", which I cannot find in the documentation at all. See above

>Also parameters "Vd(plasma)/F" and "Vss(plasma)/F", which should not appear for the iv simulation.

I don't think those parameter makes sense for oral simulation either... (at least they make sense in the global view but not for each selected output)..

**Comment by msevestre** [comment]
>Why, after calculation bioavailability, the non- /F parameters (which are labeled as /F but probably wrong...) are not shown any more?

The rest if I have explained. Here I guess we could show the /F parameters as well. not sure why we are hiding them

**Comment by msevestre** [comment]
moving this to the forum as discussion so that it can be found later on
